samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,353 @@
+
+.. _CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS:
+
+CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS - Cooldown after max consecutive send failures exceeded (seconds)
+   Sets the cooldown time (in seconds) after the maximum number of consecutive send failures is exceeded.
+   If a connection reset is triggered by too many failed device messages, the sample waits for this long (in seconds) before trying again.
+
+.. _CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS:
+
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS - Sensor sampling interval (seconds)
+   Sets the time to wait between each temperature sensor sample.
+
+.. note::
+   Decreasing the sensor sampling interval too much leads to more frequent use of the LTE connection, which can prevent GNSS from obtaining a fix.
+   This is because GNSS can operate only when the LTE connection is not active.
+   Every time a sensor sample is sent, it interrupts any attempted GNSS fix.
+   The exact time required to obtain a GNSS fix will vary depending on satellite visibility, time since last fix, the type of antenna used, and other environmental factors.
+   In good conditions, and with A-GPS data, one minute is generally just barely long enough to get a fix most of the time.
+
+   The default sensor sampling interval, 60 seconds, will quickly consume cellular data, and should not be used in a production environment.
+   Instead, consider using a less frequent interval, and if necessary, combining multiple sensor samples into a single `device message <nRF Cloud Device Messages_>`_ , or combining multiple device messages using the `d2c/bulk device message topic <nRF Cloud MQTT Topics_>`_.
+
+   If you significantly increase the sampling interval, you must keep the :kconfig:option:`CONFIG_MQTT_KEEPALIVE` short to avoid the carrier silently closing the MQTT connection due to inactivity, which could result in dropped device messages.
+   The exact maximum value depends on your carrier.
+   In general, a few minutes or less should work well.
+
+.. _CONFIG_GNSS_ANTENNA_ONBOARD:
+
+CONFIG_GNSS_ANTENNA_ONBOARD - Select the onboard GNSS antenna (default)
+   This option enables the onboard GNSS antenna.
+
+.. _CONFIG_GNSS_ANTENNA_EXTERNAL:
+
+CONFIG_GNSS_ANTENNA_EXTERNAL - Select an external GNSS antenna
+   This option enables the external GNSS antenna.
+
+.._CONFIG_GNSS_AT_MAGPIO:
+
+CONFIG_GNSS_AT_MAGPIO - ``AT%XMAGPIO`` command
+   Sets the ``AT%XMAGPIO`` command to be sent to the GNSS module.
+   Leave empty if this command should not be sent.
+   This command must be sent for GNSS to function properly.
+
+   A reasonable value (that selects the internal GNSS antenna) is automatically selected for this option based on the build target.
+   You should never have to mess with it yourself unless you are using a non-standard board or custom GNSS antenna.
+   See `nRF9160 SiP pin configuration`_ for more details.
```

Comment:
```suggestion
   Sets the ``AT%XMAGPIO`` command to be sent to the GNSS module.
   This command must be sent for GNSS to function properly.

   An appropriate value is automatically selected for this option based on the build target.
   You should not need to change it yourself unless you are using a non-standard board or a custom GNSS antenna.
   See `nRF9160 SiP pin configuration`_ for more details.
```
The value of MAGPIO does not seem to depend on GNSS_ANTENNA_ONBOARD/GNSS_ANTENNA_EXTERNAL so I would remove the reference to internal GNSS antenna from here. I believe "messing with something" is somewhat informal, sorry I used it in the first place :) Also I would remove "Leave empty if this command should not be sent." because that would be exactly the kind of fiddling we want to discourage.

Reply:
Ah, good points!